AlfrescoApp/View Controllers/Sync View Controller/Data Source/RepositoryCollectionViewDataSource.m

```diff
@@ -24,6 +24,7 @@
 #import "ThumbnailManager.h"
 #import "FavouriteManager.h"
 #import "RealmSyncManager.h"
+#import "FavoritesCollectionViewDataSource.h"
```

Comment:
This should not be necessary considering the parent-child relationship between these two classes.